data/kiojoel.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Akinsanya Joel",
+  "bio": "Web Developer | Writer | Open Source"
```

Please add the missing comma here. 
```suggestion
  "bio": "Web Developer | Writer | Open Source",
```